frontend/src/app/Settings/Facility/FacilityForm.tsx

```diff
@@ -77,6 +80,34 @@ export const useFacilityValidation = (facility: Facility) => {
   return { errors, validateField, validateFacility };
 };
 
+const createFieldError = (field: keyof FacilityErrors, facility: Facility) => {
+  // The `state` field may produce two different errors: one indicating
+  // that no option has been selected and the other indicating that
+  // SimpleReport has not gone live in that particular state.
+  let error: any;
+  if (field === "state" && stateCodes.includes(facility[field])) {
+    error = (
+      <>
+        <span>
+          SimpleReport isn’t currently supported in{" "}
+          {getStateNameFromCode(facility.state)}.
+        </span>
+        <span className="display-block margin-top-05">
+          See a{" "}
+          <a href={urls.FACILITY_INFO}>
+            {" "}
+            list of states where SimpleReport is supported
+          </a>
+          .
+        </span>
+      </>
+    );
+  } else {
+    error = allFacilityErrors[field];
+  }
+  return error;
```

Comment:
I would rewrite this as follow to avoid a mutable variable and the use of `any`. This will also require the updating of the definition of `FacilityErrors` from
`export type FacilityErrors = Partial<Record<FacilityErrorKeys, string>>;`
to
`export type FacilityErrors = Partial<Record<FacilityErrorKeys, React.ReactNode>>;`

The type change may expose some bugs/typescript compile errors as the consumers of `FacilityErrors` are all expecting a `string`

```suggestion
  if (field === "state" && stateCodes.includes(facility[field])) {
    return (
      <>
        <span>
          SimpleReport isn’t currently supported in{" "}
          {getStateNameFromCode(facility.state)}.
        </span>
        <span className="display-block margin-top-05">
          See a{" "}
          <a href={urls.FACILITY_INFO}>
            {" "}
            list of states where SimpleReport is supported
          </a>
          .
        </span>
      </>
    );
  }
  return allFacilityErrors[field];
```

Reply:
Good suggestion, thank you! This broke ManageDevices a little bit but nothing else.